marlowe-playground-client/src/Halogen/ActusBlockly.purs

```diff
@@ -39,7 +39,7 @@ type State
     , generator :: Maybe Generator
     , errorMessage :: Maybe String
     , showShiny :: Boolean
-    , hasUnsavedChanges :: Boolean
+    , useEvents :: Boolean
```

Comment:
Erm... `uiEventHasFired`? `userHasInteracted`? Meh, they're not great names either.

Reply:
I'll leave as is until inspiration strikes with something that is _definitely_ clearer